symengine/dense_matrix.cpp

```diff
@@ -809,30 +771,30 @@ void fraction_free_LDU(const DenseMatrix &A, DenseMatrix &L, DenseMatrix &D,
     for (i = 0; i < row; i++)
         for (j = 0; j < row; j++)
             if (i != j)
-                L.m_[i*col + j] = zero;
+                L.m_[i * col + j] = zero;
             else
-                L.m_[i*col + i] = one;
+                L.m_[i * col + i] = one;
 
     // Initialize D
-    for (i = 0; i < row*row; i++)
+    for (i = 0; i < row * row; i++)
         D.m_[i] = zero; // Integer zero
 
     for (k = 0; k < row - 1; k++) {
-        L.m_[k*col + k] = U.m_[k*col + k];
-        D.m_[k*col + k] = mul(old, U.m_[k*col + k]);
+        L.m_[k * col + k] = U.m_[k * col + k];
+        D.m_[k * col + k] = mul(old, U.m_[k * col + k]);
 
         for (i = k + 1; i < row; i++) {
-            L.m_[i*col + k] = U.m_[i*col + k];
+            L.m_[i * col + k] = U.m_[i * col + k];
             for (j = k + 1; j < col; j++)
-                U.m_[i*col + j] = div(sub(mul(U.m_[k*col + k], U.m_[i*col + j]),
-                    mul(U.m_[k*col + j], U.m_[i*col + k])), old);
-            U.m_[i*col + k] = zero; // Integer zero
+                U.m_[i * col + j]
```

Comment:
Opened in a text-editor,
```
for (j = k + 1; j < col; j++)
                U.m_[i * col + j]
                    = div(sub(mul(U.m_[k * col + k], U.m_[i * col + j]), mul(U.m_[k * col + j], U.m_[i * col + k])), old);
```

I think this looks good and succint as to what's happening. clang-format did this because of the the maximum column limit.